### DISCUSSION TITLE
Create release branch & GitHub release for new versions instead of announcing them via a Discussion thread

### DISCUSSION THREAD

**Discussion by Ein-Tim** [Digitaler-Impfnachweis/covpass-ios#215] [Ideas]
## Current implementation

Currently, @alexcimander creates a new discussion thread in the iOS & Android repo each time a version has been finalized and is ready for rollout. However, this announcement does not include the source code.

## Suggested enhancement

Create a GitHub release instead of a discussions thread & release the source code of the version when announcing it (create the corresponding release branch). 
To give the community room for discussion, you can [create a discussion from the release](https://github.blog/changelog/2021-04-06-releases-support-comments-and-reactions-with-discussion-linking/).

**Comment by alexcimander** [comment]
Nice idea! Will discuss it within the team 👍  However for 1.32 we prepared "the old way" already - sorry!
